<launch>
  <param name="use_sim_time" value="true"/>

  <group ns="object_detector">
    <rosparam file="$(find depth_image_extractor)/config/object_detection_drone.yaml"/>
  </group>
  <node name="object_detector" pkg="depth_image_extractor" type="detection_node" output="screen">
  </node>
</launch>